bokeh/properties.py

```diff
@@ -483,6 +483,21 @@ def validate(self, value):
     def __str__(self):
         return "%s(%r)" % (self.__class__.__name__, self.regex.pattern)
 
+class JSON(String):
+    """ JSON type property validates that text values contain valid
+    JSON.
+    """
+    def validate(self, value):
+        super(JSON, self).validate(value)
+
+        if value is None: return
+
+        try:
+            import json
+            temp = json.loads(value)
```

Comment:
`bokeh/properties.py:497:13: F841 local variable 'temp' is assigned to but never used`